controllers/operands/ssp_test.go

```diff
@@ -756,14 +756,60 @@ var _ = Describe("SSP Operands", func() {
 					goldenImageList, err := getDataImportCronTemplates(hco)
 					Expect(err).ToNot(HaveOccurred())
 					Expect(goldenImageList).To(HaveLen(3))
-					Expect(goldenImageList).To(HaveCap(4))
 
 					statusImage2Enabled := statusImage2.DeepCopy()
+					enableDict(&statusImage2Enabled.DataImportCronTemplate)
 					statusImage2Enabled.Status.Modified = true
 
 					Expect(goldenImageList).To(ContainElements(*statusImage2Enabled, statusImage3, statusImage4))
 				})
 
+				It("Should not add a common DIC template if it is disabled in the file", func() {
```

Comment:
Yeah I'm still not convinced by this behaviour, we shouldn't be marking internal DICTs as disabled and should just remove them so this just smells like something that could trip folks up with the process in the future.

Reply:
Should I remove the test?

Reply:
Yes, please do. I don't think having such DICTs in the HCO image is a valid use-case.

Reply:
ok, done